### DISCUSSION TITLE
16 settembre 2021

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/mdp2223#4] [Q&A] [answered]
In questa discussion a tema la traccia di esame del 16 settembre 2021 discuteremo di eventuali proposte di soluzione all'esame in questione

**Comment by Gio0203c** [comment, marked as answer]
[16-09-2021 - COMPLETO.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/12988504/16-09-2021.-.COMPLETO.pdf)

**Comment by Frisbon** [comment]
Senza soluzione [MDP_2021-09-16_Traccia.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/13976103/MDP_2021-09-16_Traccia.pdf)
